pkgs/development/python-modules/generic/wrap.sh

```diff
@@ -1,51 +1,82 @@
+# Wrapper around wrapPythonProgramsIn, below. The $pythonPath
+# variable is passed in from the buildPythonPackage function.
 wrapPythonPrograms() {
     wrapPythonProgramsIn $out "$out $pythonPath"
 }
 
+# Transforms any binaries generated by the setup.py script, replacing them
+# with an executable shell script which will set some environment variables
+# and then call into the original binary (which has been given a .wrapped
+# prefix).
 wrapPythonProgramsIn() {
     local dir="$1"
     local pythonPath="$2"
     local python="@executable@"
-    local i
+    local path
+    local f
```

Comment:
I believe pretty firmly that the minor costs of having a couple of extra lines of diff or one more step of git blame are outweighed by having clearly named variables.